<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NamesDialogue</class>
 <widget class="QDialog" name="NamesDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Names</string>
  </property>
  <widget class="QLineEdit" name="player_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Player1</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="player_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Player2</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="player_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Player3</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="player_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Player4</string>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Set player names</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>321</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Name is max. 10 letters or digits long.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>97</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>97</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
